backend/app/services/llm/providers/registry.py

```
@@ -63,14 +61,99 @@ def get_llm_provider(
+    result, error = instance.execute(
+        completion_config=test_config, query=test_query, resolved_input=test_text
+    )
+
+    if error:
+        print(f"❌ Error: {error}")
     else:
-        logger.error(
-            f"[get_llm_provider] Unsupported provider type requested: {provider_type}"
-        )
-        raise ValueError(f"Provider '{provider_type}' is not supported.")
+        print(f"✅ TTS Success!")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)